xilem/examples/http_cats.rs

```diff
@@ -197,8 +199,8 @@ impl Status {
             FlexSpacer::Fixed(10.),
             image,
             // TODO: Overlay on top of the image?
-            // HACK: Trailing spaces workaround scrollbar covering content
-            prose("Copyright ©️ https://http.cat      ").alignment(TextAlignment::End),
+            sized_box(prose("Copyright ©️ https://http.cat").alignment(TextAlignment::End))
+                .padding(Padding::trailing(20.)),
```

Comment:
I've tried a few different numbers for this right padding, and none are quite right. The main problem is that the images from the API have the awkward black boxes around them with duplicated information. The main https://http.cat website crops them down in a lossy way, which also isn't ideal.

Reply:
I ended up setting it to 15, so it's symmetrical with the left side. (10px for the scroll bar + 5px padding)